### ISSUE TITLE
/stop no finaliza

### ISSUE THREAD

**Issue by fedex6** [fedex6/PI-PoioHQ#1]
No finaliza el proceso pir.py porque no lo encuentra

**Comment by fedex6** [comment]
Solucionado: Habia que poner "pkill -9 -f pir.py"

**Comment by fedex6** [comment]
No, al final no es una solucion correcta porque termina el proceso y cuelga el bot

**Comment by francrubio** [comment]
Dejo acá los links que encontramos con posibles soluciones
https://stackoverflow.com/questions/20218570/how-to-determine-pid-of-process-started-via-os-system
https://stackoverflow.com/questions/32649125/obtaining-pid-of-a-shell-python-script-inside-the-script-itself

Sino , desde bash y luego scrapear:
$ pgrep script.py
$ ps aux | grep script.py

**Comment by fedex6** [comment]
Al parecer si funciona, pero el problema esta cuando finalizo la sesion de ssh.

Esto se esta poniendo feo...

**Comment by fedex6** [comment]
Hecho, finaliza ahora.
El problema del SSH hay que corregirlo, que corra como servicio y no que haya que iniciarlo.
